modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1456,6 +1456,13 @@ public void testAddObjectEntryWithObjectValidationRule() throws Exception {
 		DateTimeFormatter dateTimeFormatter = DateTimeFormatter.ofPattern(
 			"yyyy-MM-dd HH:mm");
 
+		LocalDateTime todayLocalDateTime = LocalDateTime.now();
+
+		LocalDateTime tomorrowLocalDateTime = todayLocalDateTime.plusDays(1);
```

Comment:
Hey @diogomelo09 I didn't see that there are two variables that are the same as these ones, please remove these two.